### ISSUE TITLE
Cannot use this package in Angular 5

### ISSUE THREAD

**Issue by cvazquezlos** [IonicaBizau/node-line-diff#11]
Hi there.

I am trying to use this library in my Angular 5 application because I want to compare two log entries, but I am incapable to make it works properly. I am not using File System as suggested; in its case, I am using two arrays of Strings.

More exactly, my code is:

const LineDiff = `require('../../../../node_modules/line-diff');
const str1 = ["Hola", "Que"];
const str2 = ["ola", "k"];
console.log(LineDiff(str1, str2, 0).toString());
And I get an error, so I can't execute it. The path to the package is totally correct, so that is not my mistake.

What am I doing wrong?

**Comment by IonicaBizau** [comment]
The first [two parameters should be strings](https://github.com/IonicaBizau/node-line-diff#changeoldline-addedline-sensitivity), not arrays.

Try doing something like:

```js
console.log(LineDiff(str1.join("\n"), str2.join("\n"), 0).toString());
```

**Comment by cvazquezlos** [comment]
Not working yet.

The message of the error is: 

> Uncaught (in promise): TypeError: Cannot read property 'constructor' of undefined

**Comment by IonicaBizau** [comment]
Thanks! Fixed in **`2.0.0`**.

I was wrong: the arguments can be arrays as well.
